client/src/components/GraphViewerComponent/GraphViewerComponent.js

```diff
@@ -427,11 +426,10 @@ class GraphViewerComponent extends React.Component {
     }
   }
 
-  onTwinDelete = async ids => {
```

Comment:
Why are we removing the async?

Reply:
Because I removed the async call to the `doLayout` method so now there isn't anything async in the method; nobody was actually awaiting for this method call so no other change was required.

Reply:
Ha yea good point! I am a bit rusty after the break lol!